lib/modules/datasource/conda/index.spec.ts

```diff
@@ -112,5 +252,154 @@ describe('modules/datasource/conda/index', () => {
+                  },
+                  {
+                    version: '0.0.5',
+                    createdAt: DateTime.fromISO(
+                      '2020-02-29T01:40:23Z',
+                    ).toString(),
+                    yankedReason: null,
+                  },
+                  {
+                    version: '0.0.56',
+                    createdAt: null,
+                    yankedReason: null,
+                  },
+                ],
+              },
+            },
+          },
+        });
+      httpMock
+        .scope('https://prefix.dev/api/graphql')
+        .post('')
+        .once()
+        .reply(200, {
+          data: {
+            package: {
+              variants: {
+                pages: 2,
+                page: [
+                  {
+                    version: '0.0.7',
+                    createdAt: DateTime.fromISO(
+                      '2020-02-29T01:40:21Z',
+                    ).toString(),
+                    yankedReason: null,
+                  },
+                  {
+                    version: '0.0.8',
+                    createdAt: DateTime.fromISO(
+                      '2020-02-29T01:40:20.840Z',
+                    ).toString(),
+                    yankedReason: null,
+                  },
+                  {
+                    version: '0.0.10',
+                    createdAt: DateTime.fromISO(
+                      '2020-02-29T01:40:23Z',
+                    ).toString(),
+                    yankedReason: null,
+                  },
+                  {
+                    version: '0.0.560',
+                    createdAt: null,
+                    yankedReason: null,
+                  },
+                ],
+              },
+            },
+          },
+        });
+
+      const config = {
+        packageName: 'pytest',
+        registryUrls: ['https://prefix.dev/conda-forge'],
+      };
+      const res = await getPkgReleases({
+        ...config,
+        datasource,
+      });
+      expect(res).toMatchObject({
+        registryUrl: 'https://prefix.dev/conda-forge',
+        homepage: 'https://home/url',
+        sourceUrl: 'https://dev/url',
+        releases: [
+          {
+            isDeprecated: false,
+            releaseDate: '2020-02-29T01:40:23.000Z',
+            version: '0.0.5',
+          },
+          {
+            isDeprecated: false,
+            releaseDate: '2020-02-29T01:40:21.000Z',
+            version: '0.0.7',
+          },
+          {
+            isDeprecated: false,
+            releaseDate: '2020-02-29T01:40:20.840Z',
+            version: '0.0.8',
+          },
+          {
+            isDeprecated: false,
+            releaseDate: '2020-02-29T01:40:23.000Z',
+            version: '0.0.10',
+          },
+          {
+            isDeprecated: false,
+            version: '0.0.56',
+          },
+          {
+            isDeprecated: false,
+            version: '0.0.560',
+          },
```

Comment:
```suggestion
          {
            isDeprecated: false,
            releaseTimestamp: '2020-02-29T01:40:21.000Z',
            version: '0.0.5',
          },
          {
            isDeprecated: false,
            releaseTimestamp: '2020-02-29T01:40:21.000Z',
            version: '0.0.7',
          },
          {
            isDeprecated: false,
            releaseTimestamp: '2020-02-29T01:40:20.840Z',
            version: '0.0.8',
          },
          {
            isDeprecated: false,
            releaseTimestamp: '2020-02-29T01:40:23.000Z',
            version: '0.0.10',
          },
          {
            isDeprecated: false,
            version: '0.0.56',
          },
          {
            isDeprecated: false,
            version: '0.0.560',
          },
```
the name of the field is `releaseTimestamp`

Reply:
why tsc doesn't catch this 🧐